enhancements/ingress/gateway-api-with-cluster-ingress-operator.md

```diff
@@ -266,35 +259,137 @@ The initial enhancement provides a subset of the options that are available with
 the IngressController API.  In particular, using a LoadBalancer-type service is
 supported; using a NodePort-type service or host networking is not.
 
-#### New Controller to Manage Gateway API CRDs and OpenShift Service Mesh
-
-This enhancement adds a new controller to do the following:
+#### New Controller to Manage Gateway API CRDs
 
-1. Create the gatewayclasses CRD.
-2. Reconcile GatewayClasses by installing OSSM and other CRDs when appropriate.
-
-This controller creates the gatewayclasses CRD when it starts, which enables the
-cluster admin to create a new GatewayClass.  When the cluster admin does this
-and specifies a controller name that OpenShift owns, the Ingress Operator then
-does the following:
-
-1. Install the Gateway API CRDs other than the gatewayclasses CRD, namely the gateways and httproutes CRDs.
-2. Create a Subscription to install OSSM.
-3. Create a ServiceMeshControlPlane to configure OSSM (what if it already exists?; see [Open Questions](#open-questions)).
-4. Create a LoadBalancer-type service (or OSSM may create it; see [Open Questions](#open-questions)).
-5. Create a DNSRecord for the service (see the following subsection).
+This enhancement adds a new controller to create the gatewayclasses CRD.  For
+dev preview, this controller watches for the "GatewayAPI" feature to be enabled
+in the cluster featuregate.  When the feature is enabled, this controller
+installs the Gateway API CRDs, including the gatewayclasses CRD, which enables
+the cluster admin to create a new GatewayClass.
 
 The Ingress Operator effectively owns the Gateway API CRDs.  It installs
 versions of the CRDs that are compatible with the OSSM version that it installs,
 and if the CRDs are already installed, the operator may overwrite them.
 
-When OSSM is installed, it deploys the Istio control-plane and Envoy proxy.  The
-Ingress Operator configures the ServiceMeshControlPlane to enable Gateway API
-support, which means that Istio watches Gateway and HTTPRoute objects and
-configures Envoy accordingly.  Configuring a load balancer and DNS enables
+#### New Controller to Install and Configure OpenShift Service Mesh
+
+The enhancement adds another new controller to reconcile GatewayClasses by
+installing OpenShift Service Mesh (OSSM) when appropriate.  When the cluster
+admin creates a GatewayClass that specifies the
+"openshift.io/gateway-controller" controller name, this controller then does the
+following:
+
+1. Create a Subscription to install OSSM.
```

Comment:
Can we make a note that "create a subcscription" isn't implemented in Dev Preview? (and make sure a Jira exists for this)

Reply:
What do you mean?  Creating the subscription is implemented here: https://github.com/openshift/cluster-ingress-operator/blob/a29464eda9614ac3d70c185b572f4839195e7306/pkg/operator/controller/gatewayclass/subscription.go

Reply:
My mistake.  I thought I had to run the install script because this wasn't implemented, but there was a different reason I had to run the install script.